up-and-running/auth/custom.md

```diff
@@ -0,0 +1,92 @@
+# Custom Authentication
+
+If your authentication system is not listed in [npme's supported authentication methods](/up-and-running/auth/#authentication) you may be able to use an external open-source plugin (or write your own). Here are some of the SSO plugins that exist:
```

Comment:
`Here are some of the SSO plugins that exist` -> `Here are some of the existing third-party plugins that exist`